src/desktop/apps/artwork/components/related_artworks/templates/masonry.jade

```diff
@@ -1,8 +1,12 @@
-- var columns = helpers.related_artworks.masonry(layer.artworks).columns
-include ../../../../../components/artwork_masonry/index
+if sd.ENABLE_EXPERIMENTAL_STITCH_INJECTION
+  #stitch-artwork-related-artworks-masonry
+    != sd.stitch.components.ArtworkGrid({ artworks: layer.artworks, mountId: 'stitch-artwork-related-artworks-masonry'})
+else
+  - var columns = helpers.related_artworks.masonry(layer.artworks).columns
+  include ../../../../../components/artwork_masonry/index
 
-if layer.href
-  .artwork-section__jump
-    a.avant-garde-jump-link( href= layer.href )
-      | Go to #{layer.name}
-      i.icon-chevron-small-right
+  if layer.href
```

Comment:
I think this might be indented too deep (we want this section regardless of grid use above).